### ISSUE TITLE
Missing accessibility for items in bookmark window (VoiceOver)

### ISSUE THREAD

**Issue by cyberduck** [iterate-ch/cyberduck#10853]
> **@dkocher** created the issue

> When I create bookmarks for each server, I assign a nickname to them. However, in the bookmarks table, the nicknames are not read to me with VoiceOver. Instead, I hear something like “image, FTP, Image.”

**Comment by dkocher** [comment]
In f69f52d221a8c9bdd5a391caa055580d888e97d5.
